stanford-earth/matson
# EARTH-531: Spotlight style updates.

# READY FOR REVIEW

# Summary
- Style and field updates for the spotlight content type.

# Needed By (Date)
- EOD if possible

# Steps to Test

1. Review code 
2. Check out this branch and EARTH-531 in SE3_BLT
3. Import the config `drush cim -y`
4. Clear caches
5. Create a new spotlight and fill in all fields
6. Toggle between layouts.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
